src/pages/marketplace/CampusMarketplacePlatformHub.tsx

```
@@ -0,0 +1,94 @@
+    const [selectedCategory, setSelectedCategory] = useState<string>('All');
+    const [searchQuery, setSearchQuery] = useState<string>('');
+    const [contactedItem, setContactedItem] = useState<string | null>(null);
+
+    const filtered = filterMarketplaceItems(items, selectedCategory).filter(i =>
+        i.title.toLowerCase().includes(searchQuery.toLowerCase()) ||
+        (i.courseCode && i.courseCode.toLowerCase().includes(searchQuery.toLowerCase()))
+    );
+
+    const handleContactSeller = (item: MarketplaceItem) => {
+        setContactedItem(item.title);
+        setTimeout(() => setContactedItem(null), 3000);
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)